rosboard/rosboard.py

```diff
@@ -147,6 +153,27 @@ def get_topic_qos(self, topic_name: str) -> QoSProfile:
             rospy.logwarn("QoS profiles are only used in ROS2")
             return None
 
+    def create_publisher_if_not_exists(self, topic_name:str, topic_type:str) -> None:
+        """! 
+        Function to create a ros publisher if it does not exist
+        @param topic_name (str) the topic name
+        @param topic_type (_type_) the topic type string, i.e: sensor_msgs/msg/CameraInfo
+        """
+        if topic_name in self.local_pubs:
+            return
+        self.local_pubs[topic_name] = rospy.Publisher(topic_name, self.get_msg_class(topic_type))
+
+    def publish_remote_message(self,rosboard_data: dict) ->None:
```

Comment:
`rosboard_data` is not dict right? it is a list or tuple?

Reply:
You are right. Fixed now. Im merging this